src/main/java/com/snowflake/kafka/connector/records/SnowflakeMetadataConfig.java

```diff
@@ -8,6 +8,7 @@ public class SnowflakeMetadataConfig {
   final boolean createtimeFlag;
   final boolean topicFlag;
   final boolean offsetAndPartitionFlag;
+  final boolean versionFlag;
```

Comment:
nit: `sfVersionFlag`

Reply:
Changed it to `sfConnectorVersionFlag` for consistency.